app/assets/stylesheets/views/_service_sign_in.scss

```diff
@@ -0,0 +1,29 @@
+@import "grid_layout";
+
+.service-sign-in {
+  .column-one-half { @include grid-column(1 / 2); };
+  .column-two-thirds { @include grid-column(2 / 3); };
+  .column-full { @include grid-column(1); };
+
+  h3 {
+    @include bold-24;
+    margin-bottom: 4px;
+  }
+
+  hr {
+    margin-top: 5px;
+    margin-bottom: 5px;
+  }
+
+  a {
+    text-decoration: none;
```

Comment:
Sorry, I missed this before. Links are always underlined on GOV.UK, probably good to keep it the same here. Because this all applies to all service-sign-in pages, this is changing the [self assessment page](https://www.gov.uk/log-in-file-self-assessment-tax-return/sign-in/register-self-assessment) too.